### PR TITLE
hyperthreadingMitigation slows things down way too much

### PR DESCRIPTION
**Pull request by danielparks**
Ping @genebean @kfjohnson @mikkergimenez: is this correct?

### CODE REVIEW

**Comment by genebean** [comment]
@danielparks you will want to add this too:

```
esxcli system settings advanced set -o '/UserVars/SuppressHyperthreadWarning' -i 1
```

**Comment by genebean** [comment]
Also, I have not read the rest of the script, but the disabling of the mitigation requires a reboot to take effect.

**Comment by danielparks** [comment]
Weirdly, I'm not getting warnings on the xc640s.

I added a comment about the reboot, though it's not terribly obvious. There are some things that require a reboot, and some things that don't. :p Something to clean up in the future.

(I made the changes you requested.)

File: configure-esxi-opdx-ucs-ci.sh [hunk 1]
```diff
@@ -142,11 +142,12 @@ EOF
...
 
-
-# Enable 'L1 Terminal Fault' vulnerability mitigation
+# Disable 'L1 Terminal Fault' vulnerability mitigation because it's too slow.
 # https://rakhesh.com/virtualization/what-is-esx-problem-hyperthreading-unmitigated/
+# Requires reboot.
 error_check "Invalid Key Name: hyperthreadingMitigation" \
-  esxcli system settings kernel set -s hyperthreadingMitigation -v TRUE
+  esxcli system settings kernel set -s hyperthreadingMitigation -v FALSE
+esxcli system settings advanced set -o /UserVars/SuppressHyperthreadWarning -i 1
```
Comment (genebean) [inline configure-esxi-opdx-ucs-ci.sh:150]: The quotes are required around `'/UserVars/SuppressHyperthreadWarning'`

**Review by genebean — CHANGES_REQUESTED** [review]

File: configure-esxi-opdx-ucs-ci.sh (hunk 1 above)
Comment (danielparks) [reply configure-esxi-opdx-ucs-ci.sh:150]: I just ran it on the xc640s and it worked fine — it's sh, so it shouldn't matter.

**Review by genebean — APPROVED** [review]

**Pull request merged**